docs/content/docs/development/write-table.md

```diff
@@ -0,0 +1,190 @@
+    </tbody>
+</table>
+
+## Expire Snapshot
+
+Table Store generates one or two snapshots per commit. To avoid too many snapshots
+that create a lot of small files and redundant storage, Table Store write defaults
+to eliminating expired snapshots, controlled by the following options:
+
+<table class="table table-bordered">
+    <thead>
+    <tr>
+      <th class="text-left" style="width: 20%">Option</th>
+      <th class="text-center" style="width: 5%">Required</th>
+      <th class="text-center" style="width: 5%">Default</th>
+      <th class="text-center" style="width: 10%">Type</th>
+      <th class="text-center" style="width: 60%">Description</th>
+    </tr>
+    </thead>
+    <tbody>
+    <tr>
+      <td><h5>snapshot.time-retained</h5></td>
+      <td>No</td>
+      <td style="word-wrap: break-word;">1 h</td>
+      <td>Duration</td>
+      <td>The maximum time of completed snapshots to retain.</td>
+    </tr>
+    <tr>
+      <td><h5>snapshot.num-retained</h5></td>
+      <td>No</td>
+      <td style="word-wrap: break-word;">Integer.MAX_VALUE</td>
+      <td>Integer</td>
+      <td>The maximum number of completed snapshots to retain.</td>
+    </tr>
+    </tbody>
+</table>
+
+Please note that too short retain time or too small retain number may result in:
+- Batch query cannot find the file. For example, the table is relatively large and
+  the batch query takes 10 minutes to read, but the snapshot from 10 minutes ago
+  expires, at which point the batch query will read a deleted snapshot.
+- Continuous reading jobs on FileStore (Without Log System) fail to restart. At the
+  time of the job failover, the snapshot it recorded has expired.
+
+## Performance
+
+Table Store uses LSM data structure, which itself has the ability to support a large
+number of updates. Update performance and query performance is a tradeoff, the
+following parameters control this tradeoff:
+
+<table class="table table-bordered">
+    <thead>
+    <tr>
+      <th class="text-left" style="width: 20%">Option</th>
+      <th class="text-center" style="width: 5%">Required</th>
+      <th class="text-center" style="width: 5%">Default</th>
+      <th class="text-center" style="width: 10%">Type</th>
+      <th class="text-center" style="width: 60%">Description</th>
+    </tr>
+    </thead>
+    <tbody>
+    <tr>
+      <td><h5>num-sorted-run.max</h5></td>
+      <td>No</td>
+      <td style="word-wrap: break-word;">5</td>
+      <td>Integer</td>
+      <td>The max sorted run number. Includes level0 files (one file one sorted run) and high-level runs (one level one sorted run).</td>
+    </tr>
+    </tbody>
+</table>
+
+- The larger `num-sorted-run.max`: the less merge cost when updating data, which
+  can avoid many invalid merges. However, if this value is too large, more memory
+  will be needed when merging files, because each FileReader will take up a lot
+  of memory.
+- Smaller `num-sorted-run.max`: better performance when querying, fewer files
+  will be merged.
```

Comment:
Nit: Keep consistency
- The larger `num-sorted-run.max`, the less merge cost when updating data, which
  can avoid many invalid merges. However, if this value is too large, more memory
  will be needed when merging files because each FileReader will take up a lot
  of memory.
- The smaller `num-sorted-run.max`, the better performance when querying, fewer files
  will be merged.